packages/realm/src/TypeHelpers.ts

```diff
@@ -110,6 +120,61 @@ export function mixedToBinding(realm: binding.Realm, value: unknown): binding.Mi
   }
 }
 
+//TODO NEED TO CHANGE ALSO WHAT HAPPENS ON JSI (NOT ONLY NODE)
+
+//Here I am trying to strike a balance between completeness and performance.
+//I think if we have to check everything (especially for polygon), all those type assertions will become very complex,
+//and probably also inefficient.
```

Comment:
I mean, we already sort of will be doing the assertion when converting to the binding, so the perf concerns are a bit moot. But also, see above for why I don't think you want these functions anyway.